Makefile

```diff
@@ -228,3 +234,4 @@ $(TOOLING): $(BIN_DIR)
 	@echo Installing tools from hack/tools.go
 	@cd hack/tools && go list -mod=mod -tags tools -f '{{ range .Imports }}{{ printf "%s\n" .}}{{end}}' ./ | xargs -tI % go build -mod=mod -o $(BIN_DIR) %
 	@GOBIN=$(BIN_DIR) go install $(GO_PKG)/hack/docgen
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(BIN_DIR) $(GOLANGCI_LINT_VERSION)
```

Comment:
from past experience, it'd better to pin a know version otherwise new checks could mean that suddenly our CI is blocked.

Reply:
It actually is, see `GOLANGCI_LINT_VERSION=v1.52.2` env var over there ;)

Reply:
ah my bad!